components/HistoryTable.qml

```diff
@@ -151,7 +151,7 @@ ListView {
                 anchors.leftMargin: 18 * scaleRatio
                 font.family: MoneroComponents.Style.fontLight.name
                 font.pixelSize: 14 * scaleRatio
-                text: isOut ? "Sent" : "Received"
+                text: isOut ? qsTr("Sent") + translationManager.emptyString : "Received"
```

Comment:
Why just one ?

Reply:
Because just one was noticed missing, but yes, make sense to mark both.
Donme, and resynced all files